src/matrix/inversewishart.jl

```diff
@@ -43,6 +43,16 @@ size(d::InverseWishart) = (p = dim(d); (p, p))
 params(d::InverseWishart) = (d.df, d.Ψ, d.c0)
 @inline partype{T<:Real}(d::InverseWishart{T}) = T
 
+### Conversion
+function convert{T<:Real}(::Type{InverseWishart{T}}, d::InverseWishart)
+    P = convert_eltype(T, d.Ψ)
```

Comment:
Cool. I knew there had to be some way.

These (`convert_eltype`) are new to this PR and aren't needed if there are other facilities for handling it. For instance, I can easily define `convert(ZeroVector{T}, Z)`.

Here, though, we'd want `convert(AbstractPDMat{T}, d.Ψ)` to avoid calculating the Cholesky over again, right? Should I submit a PR in PDMats for that?